google/cloud/pubsub/integration_tests/subscription_admin_integration_test.cc

```diff
@@ -34,6 +34,9 @@ namespace pubsub {
 GOOGLE_CLOUD_CPP_INLINE_NAMESPACE_BEGIN
 namespace {
 
+using ::google::cloud::pubsub::Topic;
```

Comment:
nit: this is superfluous when we are already in the `google::cloud::pubsub` namespace.

Reply:
```suggestion
```